apps/video/test/marionette/lib/video.js

```diff
@@ -192,6 +195,66 @@ Video.prototype = {
   },
 
   /**
+   * Taps 'pause' button and checks that the counter is not progressing.
+   * returns false if the counter has progressed.
+   */
+  pausePlayback: function() {
+    this.client.helper.waitForElement(Video.Selector.playpauseButton).click();
+    var currentTime = this.getElapsedTimeSeconds();
+    
+    // wait for 2 seconds
+    this.client.executeAsyncScript(function () 
+      {setTimeout(function(){
+        marionetteScriptFinished();
+      }, 2000);
+
+    });
+    return this.getElapsedTimeSeconds() == currentTime;
```

Comment:
Yes, it should fail. But why is that line necessary? Why is it necessary to wait two seconds after clicking 'pause' to determine if 'getElapsedTimeSeconds' has changed? Tests using arbitrary delays will lead to unreliable results. My comment was exploring the purpose of the delay and how to remove it.